axelrod/strategies/calculator.py

```diff
@@ -11,7 +11,13 @@ class Calculator(Player):
     """
 
     name = "Calculator"
-    memory_depth = float('inf')
+    behaviour = {
+        'memory_depth': float('inf'),
+        'stochastic': False,
```

Comment:
This one actually is stochastic, since the first 20 rounds are Joss, and it is stochastic. Looks like it was misclassified before the PR.